File: MyGroupAPI/Models/UserAttend.cs
using System;
using System.Collections.Generic;

namespace MyGroupAPI.Models
{
    public class UserAttend
    {
        public int UserAttendId { get; set; }
        public string AttendSituation { get; set; }
        public DateTime AttendDate { get; set; }
        public string ReasonOfAbsence { get; set; }
        public string Notes { get; set; }


        public User User { get; set; }
        public int UserId { get; set; }

       

        public UserAttend()
        {
            this.AttendDate = DateTime.Now;
        }

    }
}

File: MyGroupSPA/src/app/_models/user-attend.ts
import { User } from './user';

export interface UserAttend {
    userAttendId?:number;
    attendSituation?:string;
    attendDate?:Date;
    reasonOfAbsence?:string;
    notes?:string;
    user?:User;
    userId?:number;
    arabicName?:string;
    guardianName?:string;
    userGroupName?:string;
    userClassName?:string;

}
